src/ripple/app/ledger/Ledger.h

```diff
@@ -255,6 +255,12 @@ class Ledger final
         return mImmutable;
     }
 
+    bool
+    isMutable() const
```

Comment:
I understand the motivation: `if (!isImmutable())` is a double negative and reads weird. I'd be OK if `isImmutable` was replaced with `!isMutable` and we removed that function, but I'm don't love an interface that has both `isImmutable` and `isMutable`.